page/blocks/person.ejs

```diff
@@ -1,9 +1,13 @@
-<div class="col-md-6 person-info-container">
+<div class="col-md-6 person-info-container" id="<%= id %>">
     <div class="person-info">
         <img class="rounded-image" src="<%= image %>"/>
         <div class="person-details">
-            <h4 class="name"><a href="#" ><%= name %></a></h4>
-            <h4 ><%= title %></h4>
+            <h4 class="name"><a href="#" > <%= name %></a></h4>
+            <h4 class="title"><%= title %></h4>
         </div>
     </div>
+    <p class="bio hidden">
```

Comment:
Idont like this approach at all. This uses html markup only to store data that later we will read it via jquery. Lets revisit it when we will have all data in yml.